### PR TITLE
docs: Fix typos in libraries/cookies.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Fix typos in libraries/cookies.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/cookies.rst [hunk 1]
```diff
@@ -248,15 +248,15 @@ objects. However, you may wish to define your own settings by changing the follo
...
 ==================== ===================================== ========= =====================================================
-Setting              Options/ Types                        Default   Description
+Setting              Options/Types                         Default   Description
 ==================== ===================================== ========= =====================================================
 **$prefix**          ``string``                            ``''``    Prefix to prepend to the cookie name.
 **$expires**         ``DateTimeInterface|string|int``      ``0``     The expires timestamp.
 **$path**            ``string``                            ``/``     The path property of the cookie.
 **$domain**          ``string``                            ``''``    The domain property of the cookie.with trailing slash.
 **$secure**          ``true/false``                        ``false`` If to be sent over secure HTTPS.
 **$httponly**        ``true/false``                        ``true``  If not accessible to JavaScript.
```
Comment (kenjis) [inline user_guide_src/source/libraries/cookies.rst:258]:
```suggestion
**$secure**          ``true``/``false``                    ``false`` If to be sent over secure HTTPS.
**$httponly**        ``true``/``false``                    ``true``  If not accessible to JavaScript.
```

File: user_guide_src/source/libraries/cookies.rst [hunk 2]
```diff
@@ -248,15 +248,15 @@ objects. However, you may wish to define your own settings by changing the follo
...
-**$samesite**        ``Lax|None|Strict|lax|none|strict''`` ``Lax``   The SameSite attribute.
+**$samesite**        ``Lax|None|Strict``                   ``Lax``   The SameSite attribute.
 **$raw**             ``true/false``                        ``false`` If to be dispatched using ``setrawcookie()``.
```
Comment (kenjis) [inline user_guide_src/source/libraries/cookies.rst:260]:
```suggestion
**$raw**             ``true``/``false``                    ``false`` If to be dispatched using ``setrawcookie()``.
```

File: user_guide_src/source/libraries/cookies.rst [hunk 3]
```diff
@@ -248,15 +248,15 @@ objects. However, you may wish to define your own settings by changing the follo
...
+Setting              Options/Types                         Default   Description
 ==================== ===================================== ========= =====================================================
 **$prefix**          ``string``                            ``''``    Prefix to prepend to the cookie name.
 **$expires**         ``DateTimeInterface|string|int``      ``0``     The expires timestamp.
 **$path**            ``string``                            ``/``     The path property of the cookie.
 **$domain**          ``string``                            ``''``    The domain property of the cookie.with trailing slash.
 **$secure**          ``true/false``                        ``false`` If to be sent over secure HTTPS.
 **$httponly**        ``true/false``                        ``true``  If not accessible to JavaScript.
-**$samesite**        ``Lax|None|Strict|lax|none|strict''`` ``Lax``   The SameSite attribute.
+**$samesite**        ``Lax|None|Strict``                   ``Lax``   The SameSite attribute.
```
Comment (kenjis) [inline user_guide_src/source/libraries/cookies.rst:259]:
```suggestion
**$samesite**        ``Lax``/``None``/``Strict``           ``Lax``   The SameSite attribute.
```

**Review by kenjis — APPROVED** [review]

**Review by michalsn — APPROVED** [review]

**Comment by kenjis** [comment]
@obozdag Thank you!

**Pull request merged**